### PR TITLE
Feature/use twitter bootstrap

### PR DESCRIPTION
**Pull request by darly-r**
add modal window with bootstrap 3.7

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,56 +1,237 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>List of books</title>
+    <title>List of books</title>
+    <!-- <link href="node_modules\bootstrap\dist\css\bootstrap.min.css" rel="stylesheet" media="screen">
+    <script src="node_modules\jquery\dist/jquery.min.js"></script>
+    <script src="node_modules\bootstrap\dist\jsbootstrap.min.js"></script> -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```
Comment (the-daria) [inline index.html]: Please use node_modules

File: index.html [hunk 2]
```diff
@@ -1,56 +1,237 @@
...
+<h1>List of books</h1>
+<div class="table-responsive">
+<table class="table table-striped">
     <tr>
-    <td class="idClass">100005</td>
-    <td>The Girl on the Train</td>
-    <td>Paula Hawkins</td>
-    <td>£19.79</td>
-    <td><button>View details</button></td>
+        <td>
```
Comment (the-daria) [inline index.html]: Don't use table please

File: index.html (hunk 1 above)
Comment (darly-r) [reply index.html]: done

**Comment by darly-r** [comment]
1) npm modules are used
2) Tables were exchanged by grid

**Review by the-daria — APPROVED** [review]

**Pull request merged**